.github/workflows/docker-server.yml

```diff
@@ -55,6 +56,21 @@ jobs:
           build-args: |
             GITHUB_SHA_SHORT=${{ steps.build_args.outputs.github_sha_short }}
 
+      - name: Dispatch Commit image CICD
+        if: github.event_name == 'push' && !env.ACT
+        env:
+          PAT: ${{ secrets.COMMANDER_DATA_TOKEN }}
+          PARENT_REPO: temporalio/cicd-builds
+          WORKFLOW_ID: publish-stage-results.yml
+          PARENT_BRANCH: ${{ toJSON("master") }}
+          STAGE: ${{ toJSON("builds") }}
+          COMPONENT: ${{ toJSON("temporal") }}
+          TAG: sha-${{ steps.build_args.outputs.github_sha_short }}
+          AUTHOR: ${{ toJSON(github.event.pusher.name) }}
```

Comment:
the pusher isn't always the commit author, right? why not use the commit author here?

Reply:
changed to commit author